### PR TITLE
docs: Add missing required `delete_repo` permission

### PR DESCRIPTION
**Pull request by tomkerkhove**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ USAGE:
 ARGUMENTS:
     <GITHUB-API-KEY>    The GitHub API key to use.
                         The API key must have the following permissions: 
-                        public_repo, delete_packages   
+                        public_repo, delete_repo, delete_packages
```
Comment (patriksvensson) [inline README.md:25]: That is actually a typo. `delete_packages` should be deleted as well.

**Review by patriksvensson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tomkerkhove) [reply README.md]: That's what I figured but wasn't sure about!

**Review by patriksvensson — APPROVED** [review]
LGTM!

**Comment by patriksvensson** [comment]
Merged! Thank you for your contribution. Much appreciated! 👍

**Pull request merged**